erpnext/projects/doctype/project/project.py

```diff
@@ -234,6 +234,14 @@ def notify(self):
 
 			frappe.publish_realtime('show_notification_alert', message=notification_doc.get("subject"), after_commit=True, user=user)
 
+
+@frappe.whitelist()
```

Comment:
You don't need to whitelist this function. re-write the function within self

Reply:
removed whitelist function

Reply:
While accessing any server-side method via frappe.call(), you need to whitelist the server-side method using decorator @frappe.whitelist.

@meshramsaroj  don't remove frappe.whitelist